examples/trials/auto-gbdt/config.yml

```diff
@@ -0,0 +1,20 @@
+authorName: default
+experimentName: example_auto-gbdt
+trialConcurrency: 1
+maxExecDuration: 10h
+maxTrialNum: 1
+#choice: local, remote
+trainingServicePlatform: local
+searchSpacePath: ~/nni/examples/trials/auto-gbdt/search_space.json

```

Comment:
for your convenience, nni has supported relative path, you can simply use ./search_space.json :)